Dockerfile-libreoffice-jdk

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.15.0
+
+USER root
+
+RUN apk upgrade --no-cache && apk add --no-cache openjdk17 curl msttcorefonts-installer fontconfig libreoffice
+RUN update-ms-fonts
+
+USER 1000
```

Comment:
Should this be 10000?

Reply:
no because we aren't creating the hocs_user, we can but its just extra guff for a build image

Reply:
Yep just checked the other JDK one.